src/nl/jomco/apie/report.clj

```diff
@@ -248,7 +341,17 @@
     (example/example openapi (subvec canonical-schema-path 0 (dec (count canonical-schema-path))))))
 
 (defn- path->hiccup [coll]
-  [:span.path (interpose " / " (map #(vector :code %) coll))])
+  [:span.path
+   "["
+   (interpose "]["
+              (loop [[k & ks] coll
```

Comment:
Waarom niet iets als
`(interpose "][" (map (fn [k] [:code (pr-str (if (keyword? k) (name k) k)))`

En eigenlijk zou ik niet de individuele items in het path als `<code>` doen, maar het hele path (dus `[:code.path]` ipv `[:span.path]`